lib/feed/s3.rb

```diff
@@ -2,30 +2,32 @@ class Feed
   class S3
     class << self
       def setup(options)
-        AWS.config(access_key_id: options[:access_key_id], secret_access_key: options[:secret_access_key])
+        Aws.config.update(
+          access_key_id: options[:access_key_id], secret_access_key: options[:secret_access_key]
```

Comment:
What @carpodaster said about the region could also go in here from what I remember.